### DISCUSSION TITLE
token expired or invalid: 403

### DISCUSSION THREAD

**Discussion by MEdydady** [community/community#118310] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![Screenshot_20240408_155420](https://github.com/community/community/assets/45405733/6e07cf09-9ec3-48fe-a861-34cf6ff2cf70)

After reinstalling VSC, there's an issue with Copilot. The subscription is paid. Has anyone encountered this problem?  Could this be related to VPN? If yes, can it be fixed without disabling VPN?

OS - Linux Manjaro 6.6.25-1-MANJARO (64-bit)
VSC -  1.88.0 
Github Copilot Chat - v0.14.0
Github Copilot  - v1.178.0

**Comment by davecheney** [comment, marked as answer]
@MEdydady it looks like there is a billing issue with your account. 
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
